### PR TITLE
Add recipe for kr8s

### PR DESCRIPTION
**Pull request by jacobtomlinson**
Depends on #23180 and #23181

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/python-jsonpath/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+about:
+  summary: Another JSONPath implementation for Python.
+  dev_url: https://github.com/jg-rp/python-jsonpath
+  license: MIT
+  license_file: LICENSE.txt
+
+extra:
+  recipe-maintainers:
+    - jacobtomlinson
```
Comment (jacobtomlinson) [inline recipes/python-jsonpath/meta.yaml:41]: @jg-rp I see this package isn't on conda forge and I need it for my package. Are you happy with me adding this and are you ok with me adding you to this list?

File: recipes/asyncio-atexit/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+about:
+  home: https://github.com/minrk/asyncio-atexit
+  summary: Like atexit, but for asyncio
+  license: MIT
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - jacobtomlinson
```
Comment (jacobtomlinson) [inline recipes/asyncio-atexit/meta.yaml:42]: @minrk I see this package isn't on conda forge and I need it for my package. Are you happy with me adding this and are you ok with me adding you to this list?

File: recipes/python-jsonpath/meta.yaml (hunk 1 above)
Comment (jg-rp) [reply recipes/python-jsonpath/meta.yaml:41]: Yes, I'm happy. And I'm OK with being listed as a maintainer.

Comment (jacobtomlinson) [reply recipes/python-jsonpath/meta.yaml:42]: Thank you!

File: recipes/asyncio-atexit/meta.yaml (hunk 2 above)
Comment (jacobtomlinson) [reply recipes/asyncio-atexit/meta.yaml]: I've moved this into a separate PR https://github.com/conda-forge/staged-recipes/pull/23181

**Comment by jacobtomlinson** [comment]
I'm a little stuck here, builds are failing on macOS and Windows and I *think* it's related to `poetry-dynamic-versioning`. Has anyone from @conda-forge/help-python seen this before or have any ideas?

File: recipes/kr8s/meta.yaml [hunk 3]
```diff
@@ -6,8 +6,8 @@ package:
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/kr8s-{{ version }}.tar.gz
-  sha256: 0308937d0eaba89bd9b7acf976866cdf8a36568872a3263a9447293d4d14e431
+  url: https://github.com/kr8s-org/kr8s/archive/refs/tags/v{{ version }}.zip
```
Comment (ocefpaf) [inline recipes/kr8s/meta.yaml:9]: I believe you need version 0.5.1 first in dask-kubernetes, no?

Comment (ocefpaf) [reply recipes/kr8s/meta.yaml]: Using the GH source did not work :-/

**Comment by ocefpaf** [comment]
@jacobtomlinson this is noarch and the linux package, that is passing, will be installable on Windows and macOS. I'm tempted to merge this PR b/c debugging why those two archs are not building is not worth it. Are you OK with that?

**Pull request merged**